libc/test/src/sys/mman/linux/mincore_test.cpp

```diff
@@ -16,12 +16,13 @@
 #include "test/UnitTest/ErrnoSetterMatcher.h"
 #include "test/UnitTest/Test.h"
 
+#include <unistd.h>
+
 using LIBC_NAMESPACE::testing::ErrnoSetterMatcher::Fails;
 using LIBC_NAMESPACE::testing::ErrnoSetterMatcher::Succeeds;
 using LlvmLibcMincoreTest = LIBC_NAMESPACE::testing::ErrnoCheckingTest;
 
-// TODO: Replace with sysconf call once the function is properly implemented.
-constexpr size_t PAGE_SIZE = 4096;
+const size_t PAGE_SIZE = sysconf(_SC_PAGESIZE);
```

Comment:
you need to call the sysconf in the LLVM-libc namespace.
```suggestion
const size_t PAGE_SIZE = LIBC_NAMESPACE::sysconf(_SC_PAGESIZE);
```

Reply:
It's fine for tests, but just be aware that this changes the behavior to require a global constructor.

Reply:
Right, I also figured that would be fine for tests

Reply:
Done